camelot/admin/object_admin.py

```diff
@@ -485,6 +498,12 @@ def get_related_admin(self, cls):
         related_admin = self.app_admin.get_related_admin(cls)
         if not related_admin:
             logger.warn('no related admin found for %s' % (cls.__name__))
+        # Check for 'deep_editable' in __facade_args__, default is True
+        if isinstance(self.entity, EntityMeta):
```

Comment:
de idee van die 'deep not editable' laten we ook vallen, uiteindelijk weet je nooit wat dat nu exact voor gevolg heeft.

op een aantal plaatsen zullen we dit moeten oplossen door een aantal related classes ook op facade args editable=False te zetten, op andere plaatsen zullen we moeten nadenken :)